juntagrico/views_subscription.py

```diff
@@ -427,6 +427,26 @@ def cancel_extra(request, extra_id, subscription_id):
     return return_to_previous_location(request)
 
 
+@primary_member_of_subscription
+def change_nickname(request, subscription_id):
+    subscription = get_object_or_404(Subscription, id=subscription_id)
+    if subscription.nickname is None:
```

Comment:
das kann nicht funktionieren so kann der nickname nie inital gesetzt werden

Reply:
> das kann nicht funktionieren so kann der nickname nie inital gesetzt werden

My understanding was, that nicknames should initially only be enabled by admins in the data administration? That this feature should not be publicly accessible to everybody? @pascalfree

Reply:
no the idea is that the nickname is only used in the depotlist in the fasion
primarymember, nickname
and nowhere else
also we do not show the fact that no nickname is set in the subscritption overview
but in the abo change overview which is the entrypoint for all changes we add a row to change the nickname and a little explanation where it used
this way we do not need to bother the admins but we do not promote it people have to find it and it does not obscure information in relevant places